__tests__/fixtures/request-cache/GET/localhost/.bin

```diff
@@ -1,5 +1,5 @@
 HTTP/1.1 200 OK
-Date: Wed, 19 Oct 2016 13:30:48 GMT
+Date: Mon, 24 Oct 2016 13:03:20 GMT

```

Comment:
I think this should be in the `.gitignore`, but I'm not sure what the generalized entry would be.